specs/protocol/exec-engine.md

```diff
@@ -81,7 +81,7 @@ The three types of fees are collected in 3 distinct L2 fee-vault deployments for
 fee payments are not registered as internal EVM calls, and thus distinguished better this way.
 
 These are hardcoded addresses, pointing at pre-deployed proxy contracts.
-The proxies are backed by vault contract deployments, based on `FeeVault`, to route vault funds to L1 securely.
```

Comment:
I don't think that we should be too specific here on the behavior of the `FeeVault`. This is free software, it can be configured however

Reply:
Do you have a suggested wording? Can we just replace "are" with "may"?

Reply:
perhaps something like "based on FeeVault, to collect fees"

simple and not opinionated on where the fees go